src/Engine/PriceCalculator.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+#include "Engine/Objects/Items.h"
+
+class Player;
+// Class to calculate various prices and overall deal with them
+class PriceCalculator {
+ public:
+  static int getBaseIdentifyPrice(float priceMultiplier);
+  static int getBaseRepairPrice(int uRealValue, float priceMultiplier);
+  static int getBaseBuyingPrice(int uRealValue, float priceMultiplier);
+  static int getBaseSellingPrice(int uRealValue, float priceMultiplier);
+
+  static int getItemRepairPriceForPlayer(Player* player, int uRealValue,
```

Comment:
I believe here and below `Player` should be a const pointer.

Reply:
I need to const-qualify several methods in Player to do this, should I do all others that qualify along the way (in Player) or leave that for another PR?

Reply:
Or did you mean const pointer, not pointer to const player?

Reply:
Okay turns out I can't make pointer to const Player, because in long chain of methods there is finally call to Player::GetItem(), which can't return const Item I think, item should be modifiable.

Reply:
I could create const-only version though, which would return const ItemGen, and leave current method alone.

Reply:
Yeah, I'm talking about `const Player *`.

OK if it's messy we can do this in a separate PR.